qiling/os/windows/dlls/__init__.py

```diff
@@ -9,3 +9,4 @@
 from .wininet import *
 from .msi import *
 from .ntdll import *
+from .ucrtbased import *
```

Comment:
It seems CI failed because there is no `ucrtbased` module

Reply:
That was it. Just forgot to push that file, which is why I wasn't seeing the error on my side. Sorry, haven't worked with Travis much and didn't see a way to dig into the results. 

Thanks!